toolkit/tools/precacher/precacher.go

```diff
@@ -350,12 +355,12 @@ func precachePackage(pkg *repocloner.RepoPackage, packagesAvailableFromRepos map
 	const (
 		downloadRetryAttempts = 2
 		downloadRetryDuration = time.Second
+		stripEpoch            = false
+		leaveEpoch            = true
```

Comment:
Nit-pick: alphabetical order.
```suggestion
		leaveEpoch            = true
		stripEpoch            = false
```

Reply:
removed